vitest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vitest/config'
```

Comment:
Please add:
- plugin for React: `@vitejs/plugin-react` -  the tasks of this plugin are described on the page https://github.com/vitejs/vite/tree/main/packages/plugin-react#readme
- vite-tsconfig-paths - all information you can fin on this page https://github.com/aleclarson/vite-tsconfig-paths

then add plugins: [react(), tsconfigPaths()], to defineConfig

Reply:
Done.